mithril-common/src/crypto_helper/genesis.rs

```diff
@@ -18,7 +18,8 @@ pub enum ProtocolGenesisError {
 /// [Genesis Certificate](https://mithril.network/doc/mithril/mithril-protocol/certificates#the-certificate-chain-design)
 #[derive(Debug, Serialize, Deserialize)]
 pub struct ProtocolGenesisSigner {
-    pub(crate) secret_key: ProtocolGenesisSecretKey,
+    /// Protocol Genesis secret key
+    pub secret_key: ProtocolGenesisSecretKey,
```

Comment:
Instead of making this sensible data public you could add a `export_to_file(path: &Path)` method that is doc hidden. WDYT ?

Reply:
I completely agree, done :heavy_check_mark: